Flow.Launcher.Infrastructure/Http/Http.cs

```diff
@@ -34,6 +34,7 @@ static Http()
 
             client = new HttpClient(socketsHttpHandler, false);
             client.DefaultRequestHeaders.Add("User-Agent", UserAgent);
+            HttpClient.DefaultProxy = WebProxy;
```

Comment:
Not sure why but the proxy setting in `socketsHttpHandler` does not seem to work for me. Maybe you could investigate further but using this `DefaultProxy` would make it work correctly.

I did not verify if it will work if there is no proxy set.

Reply:
Hmmm that's weird, because when I test it with an unusable proxy, making changed to proxy setting do crash due to unavaliable proxy.
Thank you for the fix!
Since we will use DefaultProxy, would you please remove the socketsHttpHandler, which is useless, and make the client initialize with default parameter.

Reply:
Done!

Reply:
Thank you!